R/knitr-methods.R

```diff
@@ -20,6 +20,9 @@ registerMethods <- function(methods) {
 }
 
 .onLoad <- function(...) {
+  # warn if the version of shiny is lower than what was specified in DESCRIPTION
+  checkShinyVersion(error = FALSE)
```

Comment:
R CMD check is not that smart :) It cannot detect `packageStartupMessage` when it is called from another function.